ui/vic-ui-h5c/vic/src/vic-app/src/app/summary-view/summary-view.component.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ Copyright 2017 VMware, Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { GlobalsService, RefreshService } from '../shared/index';
+import { DataPropertyService } from '../services/data-property.service';
+import { Subscription } from 'rxjs/Rx';
+
+@Component({
+    selector: 'vic-summary-view',
+    styleUrls: ['vic-summary-view.scss'],
+    template: `
+    <div class="summary-container">
+        <div class="col-xs-12">
+            <img class="vic-summary-view-logo" src="{{vicLogoPath}}">
+            <ul class="summary-items-list">
+                <li>
+                    <span class="summary-label">
+                        Vendor
+                    </span>
+                    <span class="summary-value">
+                        VMware
+                    </span>
+                </li>
+                <li>
+                    <span class="summary-label">
+                        Version
+                    </span>
+                    <span class="summary-value">
+                        {{ pluginVersion }}
+                    </span>
+                </li>
+                <li>
+                    <span class="summary-label">
+                        Virtual Container Hosts
+                    </span>
+                    <span class="summary-value">
+                        {{ vchVmsLen }}
+                    </span>
+                </li>
+            </ul>
+        </div>
+    </div>
+    `
+})
+
+export class VicSummaryViewComponent implements OnInit, OnDestroy {
+    public vicLogoPath: string;
+    public pluginVersion: string;
+    public vchVmsLen: number;
+    private rootInfoSubscription: Subscription;
+    private refreshSubscription: Subscription;
+
+    constructor(
+        private gs: GlobalsService,
+        private refreshService: RefreshService,
+        private dataPropertyService: DataPropertyService
+    ) {
+        this.vicLogoPath = this.isPluginMode() ?
+            this.gs.getWebContextPath() + '/assets/vic-icons/100x100.png' :
```

Comment:
These types of paths that may be used elsewhere should be set as a const probably in a constant file